metadata_tools/hosts/GO_0xxx/GO_0xxx_geometry_cloud.py

```
@@ -0,0 +1,74 @@
+#########################################################################################
+import asyncio
+import sys
+from cloud_tasks.worker import Worker, WorkerData
+
+import metadata_tools.util as util
+import metadata_tools.common as com
+import host_config as hconf
+import geometry_config as config
+from metadata_tools.geometry_support import process_tables, get_args
+
+#========================================================================================
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)